pkg/controller/operators/olm/operator.go

```diff
@@ -310,6 +314,34 @@ func (a *Operator) syncObject(obj interface{}) (syncError error) {
 		a.requeueOwnerCSVs(metaObj)
 	}
 
+	namespace, ok := obj.(*corev1.Namespace)
+	if ok {
```

Comment:
nit: if you invert this to:
```go
if !ok {
    return nil
}
```
You should be able to un-indent the logic below.

Reply:
Fixed this.